src/main/java/duke/storage/OpenCsv.java

```diff
@@ -0,0 +1,49 @@
+package duke.storage;
+
+import com.opencsv.CSVWriter;
+
+import duke.exceptions.DukeException;
+import duke.models.Locker;
+
+import java.io.Writer;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.io.IOException;
+import java.util.List;
+
+public class OpenCsv {
+    private static final String STRING_ARRAY_SAMPLE = "data.csv";
```

Comment:
I think we should name it something like ```private static final String CSV_OUTPUT_PATH```

Reply:
resolved